pkg/app/piped/cmd/piped/piped.go

```diff
@@ -605,7 +605,8 @@ func (p *piped) sendPipedMeta(ctx context.Context, client pipedservice.Client, c
 	}
 
 	for retry.WaitNext(ctx) {
-		if _, err = client.ReportPipedMeta(ctx, req); err == nil {
+		if res, err := client.ReportPipedMeta(ctx, req); err == nil {
```

Comment:
I was able to retrieve get `res` when I checked in the local environment. Therefore, I checked the empty field as follows.
![Screen Shot 2021-09-29 at 16 33 00](https://user-images.githubusercontent.com/59436572/135223288-0fb45b2f-b6f5-46e1-b174-66b465424bb6.png)

Reply:
I think it was empty because currently PipedMeta is called after the notifier was started.
You can check at

1. https://github.com/pipe-cd/pipe/blob/a0cea27d61c03f2980f8d31e25c4960c7c919426/pkg/app/piped/cmd/piped/piped.go#L194-L198
2. https://github.com/pipe-cd/pipe/blob/a0cea27d61c03f2980f8d31e25c4960c7c919426/pkg/app/piped/cmd/piped/piped.go#L142-L149
3. https://github.com/pipe-cd/pipe/blob/a0cea27d61c03f2980f8d31e25c4960c7c919426/pkg/app/piped/notifier/notifier.go#L102-L108

So I think we have to move 2 to after 1.